contracts/governance/liquidation/PriceOracleLiquidationStrategy.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.6;
+
+
+import {LiquidationStrategy} from "./LiquidationStrategy.sol";
+import {IPriceOracle} from "../../interfaces/IPriceOracle.sol";
+import {IERC20Ext} from "@kyber.network/utils-sc/contracts/IERC20Ext.sol";
+
+
+/// Liquidation strategy that uses Price Oracle to liquidate tokens
+/// Liquidator will receive a premimum for every transaction that they liquidate tokens in the fee pool
```

Comment:
How / where does the liquidator withdraw his premium?

Reply:
the premium is already accounted by reducing the min return dest amount right?

Reply:
Yes, but does the liquidator need to withdraw the tokens, or is it sent automatically to him

Reply:
no, I mean for example from price oracle, you can swap 1 eth to 1000 knc, however, given 5% premium, he can swap 1 eth to 950 knc and keep 50 knc as his reward

Reply:
Ahhh ok got it